src/SDKs/Resource/Management.ResourceManager/Microsoft.Azure.Management.ResourceManager.csproj

```diff
@@ -4,7 +4,7 @@
     <PackageId>Microsoft.Azure.Management.ResourceManager</PackageId>
     <Description>Provides resource group and resource management capabilities for Microsoft Azure.</Description>
     <AssemblyName>Microsoft.Azure.Management.ResourceManager</AssemblyName>
-    <VersionPrefix>1.5.0-preview</VersionPrefix>
+    <VersionPrefix>2.0.0-preview</VersionPrefix>
```

Comment:
@luanshixia you are still in preview, so no need to update the major version to 2.0.0-preview
you can continue using minor version bump.